04.ls/ls.rb

```diff
@@ -25,49 +25,49 @@ def main
   argv = opt.parse(ARGV)
 
   path = argv[0] || './'
-  corrected_path = path.end_with?('/') ? path : "#{path}/"
-  FileTest.directory?(corrected_path) or return
-  file_names = Dir.children(corrected_path).sort
+  FileTest.directory?(path) or return
+  file_names = Dir.children(path).sort
```

Comment:
```suggestion
  FileTest.directory?(path) or return
  
  file_names = Dir.children(path).sort
```

returnする可能性があるなら改行を入れて「ここでプログラムが終わる可能性があるよ」というのをアピールしてもいいかも